Sources/Commands/SwiftTestCommand.swift

```diff
@@ -966,6 +966,9 @@ final class TestRunner {
             args.append(runnerPath.pathString)
             args.append(contentsOf: runner.extraCLIOptions)
             args.append(testPath.relative(to: localFileSystem.currentWorkingDirectory!).pathString)
+            if runner.path?.components.last == "wasmkit" {
+                args.append("--")
```

Comment:
> wasmkit expects executable arguments to be passed after -- delimiter.

That has been fixed by https://github.com/swiftwasm/WasmKit/pull/175, and has been shipped as 0.1.4. So if you're going to merge https://github.com/swiftlang/swift/pull/80820, you don't need to add the `--` delimiter.